test/test_config_emcee.yml

```diff
@@ -0,0 +1,124 @@
+# These parameters are accessible to all stages
```

Comment:
Is this file different from `test/test_config.yml`? If not, can you remove `test/test_config.yml` so we only keep one?

Reply:
The differences are minor: in `test/test_config_emcee.yml`, B-purification is turned on, \ell_{max}=300 and the `emcee` sampler is chosen. I will remove `test/test_config.yml` for simplicity.